src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -315,7 +315,16 @@ public void jobLogic () {
                 } catch (Exception e) {
                     status.fail(String.format("Error uploading/copying deployment bundle to s3://%s", s3Bucket), e);
                 }
-
+            }
+            
+            // Now that the build was successful, update Pelias
```

Comment:
Do you mean the deployment?

Reply:
It used to be deployment, but was changed to build so that the job is always fired no matter the type of deployment (since each deployment type called `return`)